helm/files/conf.yaml

```diff
@@ -0,0 +1,39 @@
+{{ if .Values.assets -}}
```

Comment:
Why is this transformation required? You can take the structure as is without conversions.

Reply:
as we discussed, I'll leave this for now